src/config.ts

```diff
@@ -0,0 +1,23 @@
+import * as config from "dotenv";
+
+config.config({ path: `envs/.env.${process.env.STAGE}` });
+
+Object.keys(config).forEach(key => {
+  process.env[key] = config[key];
+});
```

Comment:
FIXME: ここだけ以下のエラーを解消したらマージする
```
import config
Element implicitly has an 'any' type because expression of type 'string' can't be used to index type 'typeof import("/Users/nasatambourine.inc/Documents/dialogflow-bot/node_modules/dotenv/types/index")'.
  No index signature with a parameter of type 'string' was found on type 'typeof import("/Users/nasatambourine.inc/Documents/dialogflow-bot/node_modules/dotenv/types/index")'.ts(7053)
Peek Problem (⌥F8)
```

Reply:
[fixed them](https://github.com/sizukutamago/dialogflow-bot/pull/18/commits/44e16959622dafa01caccf3f93a634cc1f3f41c4)